doc/HotWordBoosting-Examples.rst

```diff
@@ -0,0 +1,56 @@
+Hot-word boosting API Usage example
+===================================
+
+With DeepSpeech 0.9 release a new API feature was introduced that allows boosting probability from the scorer of given words. It is exposed in all bindings (C, Python, JS, Java and .Net). 
+
+Currently, it provides three methods for the Model class:
+
+- ``AddHotWord(word, boost)``
+- ``EraseHotWord(word)`` 
+- ``ClearHotWords()``
+
+Exact API binding for the language you are using can be found in API Reference.
+
+General usage
+-------------
+
+It is worth noting that boosting non-existent words in scorer (mostly proper nouns) or a word that share no phonetic prefix with other word in the input audio don't change the final transcription. Additionally, hot-word that has a space will not be taken into consideration, meaning that combination of words can not be boosted and each word must be added as hot-word separately. 
+
+Adjusting the boosting value
+----------------------------
+
+For hot-word boosting it is hard to determine what the optimal value that one might be searching for is. Additionally, this is dependant on the input audio file. In practice, as it was reported by DeepSpeech users, the value should be not bigger than 20.0 for positive value boosting. W Nevertheless, each usecase is different and you might need to adjust values on your own.
+
+There is a user contributed script available on separate repository for adjusting boost values:
+
+`https://github.com/Ideefixze/deepspeech-hot-words-booster <https://github.com/Ideefixze/deepspeech-hot-words-booster>`_.
```

Comment:
I hope you are going to maintain that :). Maybe in the future you would want to make that closer to upstream, although we don't have a proper dev story here for the moment and including into the main repo might be complicated. Hopefully we will find something like DeepSpeech-examples.

Maybe, now that I am writing this, maybe `DeepSpeech-examples` would be a good home for this kind of code, and you could ensure maintenance and visibility for your work, as well as ensure people can rely on it in the future?

Reply:
Yes, the ``DeepSpeech-examples`` repo would be nice place to have it. I placed it in my own as I was instruced to do that on the forums :) Soon I'll put it there.
No problem for me to keep it maintained, as it is relatively small code.

Reply:
> Yes, the `DeepSpeech-examples` repo would be nice place to have it. I placed it in my own as I was instruced to do that on the forums :) Soon I'll put it there.
> No problem for me to keep it maintained, as it is relatively small code.

If you prefer, start by sending a PR against `DeepSpeech-examples` and update this PR accordingly?